proposals/wgs/wg-governance.md

```diff
@@ -0,0 +1,219 @@
+# Work Group(WG) Roles and Organizational Governance
```

Comment:
The doc uses "work group" here and "working group" later, should probably use only one form for consistency